src/DataCite.jl

```diff
@@ -3,35 +3,27 @@ end
 
 base_url(::DataCite) = "https://api.datacite.org/works/"
 
-function description(repo::DataCite, mainpage)
-    attributes = mainpage["attributes"]
-    desc = attributes["description"]
-    authors = join.([[names[2] for names in value] for value in attributes["author"]], " ")
-    author = format_authors(authors)
-    license = attributes["license"]
-    date = attributes["published"]
-    dataset_cite = citation_text(mainpage["id"])
-    
+description(repo::DataCite, mainpage) = (mainpage["attributes"]["description"] != nothing? mainpage["attributes"]["description"] : missing)
```

Comment:
https://api.datacite.org/works/10.5061/dryad.74699 `description` is `null`

Reply:
Ah right, JSON being a subset of javascript has an actual `null`.
In 0.7 this code can be written `something(mainpage["attributes"]["description"], missing)`
For now this will do, or `Compat` could be added as a dependency to give that function.